blank_slate_ubi.ipynb

```diff
@@ -57,11 +55,8 @@
     "        \"simulation.reported_state_income_tax\": {\n",
     "            \"year:2023:10\": True\n",
     "        },\n",
-    "        \"gov.simulation.labor_supply_responses.elasticities.income\": {\n",
-    "            \"year:2023:10\": INCOME_ELASTICITY\n",
-    "        } if include_lsrs else {},\n",
-    "        \"gov.simulation.labor_supply_responses.elasticities.substitution.all\": {\n",
-    "            \"year:2023:10\": SUBSTITUTION_ELASTICITY\n",
+    "        \"gov.contrib.cbo.labor_supply.elasticities\": {\n",
```

Comment:
Set the values instead from the CBO central reform. We will be removing this bool